lib/pages/chat_list/slidable_chat_list_item.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:fluffychat/pages/chat_list/chat_list.dart';
+import 'package:fluffychat/pages/chat_list/chat_list_view_style.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_slidable/flutter_slidable.dart';
+
+class SlidableChatListItem extends StatelessWidget {
+  const SlidableChatListItem({
+    super.key,
+    required this.controller,
+    required this.slidables,
+    required this.chatListItem,
+  });
+
+  final ChatListController controller;
+  final List<Widget> slidables;
+  final Widget chatListItem;
+
+  @override
+  Widget build(BuildContext context) {
+    return Slidable(
+      // Slidables must have the same groupTag for SlidableAutoCloseBehavior to work properly
+      groupTag: 'slidable_list',
```

Comment:
i think you can create a const for both place

Reply:
This property is used to keep only 1 `Sliedable` open at the same time.
For more information, check these references:
API document: 
- https://pub.dev/documentation/flutter_slidable/latest/flutter_slidable/Slidable/groupTag.html
Issue:
- https://github.com/letsar/flutter_slidable/issues/296